### PR TITLE
Add german translation

### PR DESCRIPTION
**Pull request by Sematre**
I really like your project and had some spare time.

### CODE REVIEW

File: src/translations/de.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+    "en": {
```
Comment (Kinsiinoo) [inline src/translations/de.json:2]: @Sematre That should be "de". 😉

**Review by Kinsiinoo — CHANGES_REQUESTED** [review]

**Comment by Sematre** [comment]
Ups :facepalm:

**Review by kkoomen — APPROVED** [review]

**Comment by kkoomen** [comment]
@Sematre You forgot to add the language to the README, but I'll add that. Thanks for taking the time to add the German translation.

**Comment by kkoomen** [comment]
I noticed that this PR didn't add the key inside `src/constants.py`. I've just added that in the latest commit [01b45a0](https://github.com/kkoomen/qbr/commit/01b45a0b84f8037a1c33db9f0cb7b782e6013f0f).

**Pull request merged**